### DISCUSSION TITLE
CarPC Wardriving?

### DISCUSSION THREAD

**Discussion by turnkit** [tjaworski/AceMagic-S1-LED-TFT-Linux#40] [General]
You think this thing would work well as a wardriving rig? 

Is the psu 12v?

What’s the best way to use the display so it doesn’t burn in?

Would you consider turning on the discussions GitHub option so I don’t make all this noise here?

**Comment by tjaworski** [comment]
- I don't see why not.
- yes, 12.0V 4.0A 48.0W
- don't leave anything in the same spot for too long, rotate different screens.
- sure, I'll turn on the discussion, I'm also on reddit r/s1panel
